src/Plugin.FilePicker/Plugin.FilePicker.Abstractions/FileData.cs

```diff
@@ -25,16 +25,28 @@ public FileData(string filePath, string fileName, Func<Stream> streamGetter, Act
             _streamGetter = streamGetter;
         }
 
+        /// <summary>
+        /// Completely reads all bytes from the input stream and returns it as byte array. Can be
+        /// used when the returned file data consists of a stream, not a real filename.
+        /// </summary>
+        /// <param name="input">input stream</param>
+        /// <returns>byte array</returns>
+        public static byte[] ReadFully(Stream input)
```

Comment:
Why add a new (static) method and not just use the property that is in place for this?

Reply:
The ReadFully() method is used by the DataArray property and by the method readFile() in IOUtil.cs. The patch originally is from ArtjomP, so I guess he wanted to save some code.